server/database/queries/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,4 @@
+const connection = require('../../config/connection');
+
+exports.getSpecificCohort = (id) =>
+  connection.query('select * from cohort where id = $1', [`${id}`]);
```

Comment:
Is not necessary to use back-tick here
```suggestion
  connection.query('select * from cohort where id = $1', [ id ]);
```